<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Choose</class>
 <widget class="QWidget" name="Choose">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>494</width>
    <height>523</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(92, 167, 242);</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>150</y>
     <width>281</width>
     <height>241</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="snowman_button">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
font: 14pt &quot;Courier New&quot;;</string>
      </property>
      <property name="text">
       <string>Снеговик</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="tree_button">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
font: 14pt &quot;Courier New&quot;;</string>
      </property>
      <property name="text">
       <string>Ёлка</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="ice_button">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
font: 14pt &quot;Courier New&quot;;</string>
      </property>
      <property name="text">
       <string>Снежинка</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>118</x>
     <y>20</y>
     <width>271</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 24pt &quot;Courier New&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Что будем рисовать?</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
